index.js

```diff
@@ -82,6 +128,38 @@ module.exports = {
     return tree;
   },
 
+  postBuild({ directory }) {
+    if (!process.env.EMBER_CLI_ELECTRON) {
+      return tree;
+    }
+
+    //
+    // electron-forge needs our package.json to have a main entry pointing to
+    // electron.js. I can't figure out how to get broccoli to edit JSON for us,
+    // so we just do it manually once the build is complete.
+    //
+    //
+    let packageJsonPath = path.join(directory, 'package.json');
+    let jsonMain = path.join('electron', 'electron.js');
+    return readJson(packageJsonPath)
+      .then((json) => {
+        if (json.main === jsonMain) {
+          // In some cases, modifying this file seems to cause a rebuild, so
+          // in order to avoid an infinite loop, do nothing if there's nothing
+          // to do!
+          return;
+        }
+
+        // On linux, package.json is a sym link at this point, so we need to
+        // unlink it before writing back to it, or else we'll end up modifying
+        // the file it points to, i.e. our project's package.json!
+        return unlink(packageJsonPath).then(() => {
+          json.main = path.join('electron', 'electron.js');
+          return writeJson(packageJsonPath, json, { spaces: 2 });
```

Comment:
While we're at it, let's consider doing the same [for config](https://github.com/electron-userland/electron-forge#configuring-package)

```
json.config = json.config || {};
json.config.forge = "./resources/forge.json"
```

And replacing [`_ensurePackageJsonConfiguration`](https://github.com/felixrieseberg/ember-electron/blob/master/blueprints/ember-electron/index.js#L61-L85) with something that writes `packageJson.config.forge` to `ember-electron/forge.json`